runner/src/server/plugins/engine/components/WebsiteField.ts

```diff
@@ -69,20 +69,13 @@ export class WebsiteField extends TextField {
         const options: any = this.options;
         const {prefix} = options;
         const viewModelPrefix = {prefix: {text: prefix}};
-        const viewModel = {
+        return {
             //@ts-ignore
             ...super.getViewModel(formData, errors),
```

Comment:
Does `super.getViewModel()` now account for the value of `hide_title`? If not won't we lose that functionality?

Reply:
@srh-sloan 

Previously what happened was
![image](https://github.com/user-attachments/assets/c04197e3-5bac-4782-b754-e1d570b4a7d7)

What we have done in the website url component is once title needs to be hidden we have overridden the labal & hint both to get the same html label 

With this change what we have done is we are going to use hint as a label 

![image](https://github.com/user-attachments/assets/bd8fe750-3121-483a-9dd3-49d54b8c5b47)